### PR TITLE
Update config to handle multiple chains, and add migrations

### PR DESCRIPTION
**Pull request by b-pmcg**

### CODE REVIEW

File: migrations/057-arbitrum-all-current-votes.sql [hunk 1]
```diff
@@ -0,0 +1,52 @@
+DROP FUNCTION IF EXISTS api.all_current_votes(CHAR);
+
+--testing to replace 38
+CREATE OR REPLACE FUNCTION api.all_current_votes(arg_address CHAR)
+RETURNS TABLE (
+	poll_id integer,
+	option_id_raw character,
+	option_id integer,
```
Comment (rafinskipg) [inline migrations/057-arbitrum-all-current-votes.sql:8]: @b-pmcg  can you allow to return all the votes without grouping by pollId? 

I want to have all the history of votes and edits of a user, so in the portal we can limit for how many times a user voted on a poll, and also show the history

File: config.js [hunk 2]
```diff
@@ -72,64 +116,76 @@ const goerli = {
...
   },
+  onStart: (services) =>
+    console.log(`Starting with these services: ${Object.keys(services)}`),
 };
 
-const mainnet = {
+// const kovan = {
+//   startingBlock: 5216304,
+//   extractors: [
+//     ...makeRawLogExtractors([
```
Comment (rafinskipg) [inline config.js:137]: delete commented code

File: config.js [hunk 3]
```diff
@@ -160,28 +216,65 @@ const mainnet = {
...
-if (process.env.VL_CHAIN_NAME === "mainnet") {
-  console.log("Using mainnet config");
-  config = mainnet;
-} else if (process.env.VL_CHAIN_NAME === "kovan") {
-  console.log("Using kovan config");
+if (process.env.VL_CONFIG_NAME === 'multiple') {
+  console.log('Using multi config');
+  config = [goerli, arbitrum_v2];
+  // config = [mainnet_v2, arbitrum_v2];
+} else if (process.env.VL_CONFIG_NAME === 'kovan') {
```
Comment (rafinskipg) [inline config.js:270]: delete all kovan references

File: migrations/056-arbitrum-polling-voted-event.sql [hunk 4]
```diff
@@ -0,0 +1,18 @@
+-- update polling.voted_event table to have chain_id
+ALTER TABLE polling.voted_event
+ADD COLUMN chain_id integer;
+
+-- copied from 002, includes updates from 021
+CREATE TABLE polling.voted_event_arbitrum (
+  id              serial primary key,
+  voter           character varying(66) not null,
+  poll_id         integer not null,
+  option_id       integer,
```
Comment (rafinskipg) [inline migrations/056-arbitrum-polling-voted-event.sql:10]: i don't think we need option_:id anymore, only the raw one

File: migrations/058-arb-all-current-votes-b.sql [hunk 5]
```diff
@@ -0,0 +1,60 @@
+DROP FUNCTION IF EXISTS api.all_current_votes(CHAR);
+
+--testing to replace 38
+CREATE OR REPLACE FUNCTION api.all_current_votes(arg_address CHAR)
+RETURNS TABLE (
+	poll_id integer,
+	option_id_raw character,
+	option_id integer,
+	block_timestamp timestamp with time zone,
```
Comment (rafinskipg) [inline migrations/058-arb-all-current-votes-b.sql:9]: what's the difference between this and the other allCurrentVotes query?

Comment (b-pmcg) [reply migrations/058-arb-all-current-votes-b.sql:9]: each migration replaces the previous functions, the newest one is `060-arb-all-current-votes-c.sql`, so the ones before it are obsolete.

Comment (rafinskipg) [reply migrations/058-arb-all-current-votes-b.sql:9]: can't we just delete them? since when we merge the PR it will only use the latest one

**Pull request merged**